packages/debian/entrypoint.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+
+set -e
+set -u
+set -o pipefail
+
+
+# Build a DEB from the source files, using debuild.
+builddeb() {
+    set -x
+
+    mkdir -p /debbuild/build-pkg/src
+    cp -r /debbuild/pkg-meta/* /debbuild/build-pkg
+    cp -r /debbuild/pkg-src/* /debbuild/build-pkg/src
+
+    # Overwrite version if supported
+    sed -i 's/_VERSION_/'"${VERSION}"'/g' "/debbuild/build-pkg/debian/changelog"
+
+    pushd /debbuild/build-pkg 1>/dev/null
+    debuild -b -uc -us
+
+    cp -r /debbuild/*.deb /debbuild/result
```

Comment:
Not sure you need `-r`, except if you're trying to copy directories named `*.deb`. :)

Reply:
nop, indeed! :) 
Thanks